src/table/virtualized-table/Body.tsx

```diff
@@ -33,6 +33,7 @@ const Body = forwardRef<BodyRef, BodyProps>((props, ref) => {
   const { layout, model, theme, viewService } = useContextSelector(TableContext, (value) => value.baseProps);
   const columnWidths = useContextSelector(TableContext, (value) => value.columnWidths);
   const initialDataPages = useContextSelector(TableContext, (value) => value.initialDataPages);
+  const maxNbrLines = viewService.viewState?.maxLineCount || undefined;
```

Comment:
wont `maxNbrLines` be undefined without `|| undefined`, when `viewState` is undefined?

Reply:
Just in case `maxLineCount` is set to zero, I want to it is set to the default value which is defined in another file.

Reply:
Probably we can ignore that case.